lib/b3fs/src/lib.rs

```diff
@@ -48,3 +49,6 @@ pub mod hasher;
 
 #[cfg(test)]
 pub mod test_utils;
+
+#[cfg(not(feature = "async"))]
```

Comment:
Same here... i think it is not need to `not` feature flag

Reply:
It's needed because we only want to include the `sync` module, whenever the `async` feature is disabled.